### ISSUE TITLE
[GSoD2020] Empty Pages

### ISSUE THREAD

**Issue by manjotsidhu** [CircuitVerse/Interactive-Book#530]
## Describe the bug

In GSoD2020 there are various empty pages which are present and are shown in navigation. Here is the full list of the incomplete pages -

1. https://betalearn.circuitverse.org/docs/comb-ssi/symbols.html
2. https://betalearn.circuitverse.org/docs/comb-ssi/truth-tables.html
3. https://betalearn.circuitverse.org/docs/comb-ssi/time-behaviour.html
4. https://betalearn.circuitverse.org/docs/comb-lsi/plds.html
5. https://betalearn.circuitverse.org/docs/seq-ssi/time-diagrams.html
6. https://betalearn.circuitverse.org/docs/seq-ssi/synchronous-systems.html
7. https://betalearn.circuitverse.org/docs/seq-ssi/asynchronous-systems.html

## To Reproduce

1.  Go to any of the links above
4.  Result = Completely Empty Page

## Expected behavior

Either the pages should be completed or we have to remove it from showing in the site for now.

**Comment by manjotsidhu** [comment]
@satu0king @tachyons need inputs on how we should deal with this ?

**Comment by manjotsidhu** [comment]
I am marking these pages as drafts as we don't want to show empty pages in deployments. CC @satu0king @Shivansh2407

**Comment by Shivansh2407** [comment]
The better option would be to either remove these or just hide these pages (Hiding would help us keep a track of the empty pages and only the content needs to be added which can be done in the near future), from the Betalearn as well, so that we do not show empty pages on the Main interactive book upon the final merge👍
Keep Contributing 🎉😄

**Comment by manjotsidhu** [comment]
Yes, In jekyll if we add front-matter tag as "published: false" in the post, we can hide it from getting them published. And that's what I was planning
ref: https://www.hongkiat.com/blog/jekyll-draft/

**Comment by Shivansh2407** [comment]
Sure, Go ahead and Make a PR! We can then finalize the Interactive Book and merge it👍

**Comment by Palaksharma23** [comment]
I would love to work on this issue , can i take this up ?

**Comment by Saikirannaidu225** [comment]
Please assign me this issue. I would like to work on this issue

**Comment by vashukashyap** [comment]
Can you assign this issue to me please .I would like to work on this.
